handbook/best-practice/do-not-disturb.md

```diff
@@ -0,0 +1,20 @@
+# Do Not Disturb Mode
+
+As developers it is important to be able to focus on the code we are writing.
+This requires being free from unnecessary distractions.
+
+The way we signal to the rest of the team that we are in such a focus session is
+by turning on Slack's `Do not disturb` mode.
+
+When this status is turned on, people who try to send us a Slack message receive
+a notification saying we are in `Do not disturb` mode and asking whether they
+want to send a notification despite this.
+
+In the rare circumstances where a pressing urgent matter needs immediate
+attention, we ask for a notification to be sent anyway. Otherwise we make a note
```

Comment:
I personally use `/remind me to message this person in 1 hour` functionality in slack as a reminder.